docs/aws.md

```diff
@@ -147,7 +147,7 @@ aws route53 list-hosted-zones | jq '.HostedZones[] | select(.Name=="kubernetes.c
  ```
  aws route53 change-resource-record-sets \
   --hosted-zone-id <parent-zone-id> \
-  --change-batch subdomain.json
+  --change-batch file://subdomain.json
```

Comment:
Some digging shows this command has required the `file://` invocation pattern since 2014: https://github.com/aws/aws-cli/blob/develop/awscli/examples/route53/change-resource-record-sets.rst. So perhaps it never worked without the prefix?